### PR TITLE
some stuff

### PR DESCRIPTION
**Pull request by ZenoABC**
## Description

moves flags (intent, permissions, systemchannelflags) into separate file
and an issue fix, probably

## Checklist

- [ ] - All code changes have been tested, if any
- [ ] - The documentation has been updated accordingly for any code changes
- [ ] - This PR fixes an already existing issue
- [ ] - This PR adds a new feature (new functions, methods, classes, parameters, etc.)
- [ ] - This PR fixes a bug
- [ ] - This PR is a breaking change (removes/renames functions, parameters etc.)
- [ ] - This PR is NOT a code change (changes Documentation files, README.md, CONTRIBUTING.md etc.)

### CODE REVIEW

File: EpikCord/__init__.py [hunk 1]
```diff
@@ -31,6 +31,7 @@
 from .status_code import *
 from .thread import *
 from .type_enums import *
+from .flags import Intents, Permissions, SystemChannelFlags
```
Comment (TheUntraceable) [inline EpikCord/__init__.py:34]: Why not change this to a star import?

File: examples/slash_commands.py [hunk 2]
```diff
@@ -1,13 +1,13 @@
 import asyncio
 import datetime
 import time
-
+import humanize
```
Comment (TheUntraceable) [inline examples/slash_commands.py:4]: Where is this used?

**Review by TheUntraceable — APPROVED** [review]

File: EpikCord/__init__.py (hunk 1 above)
Comment (ZenoABC) [reply EpikCord/__init__.py:34]: My style.

File: examples/slash_commands.py (hunk 2 above)
Comment (ZenoABC) [reply examples/slash_commands.py:4]: In code, precisedelta

File: EpikCord/__init__.py (hunk 1 above)
Comment (ZenoABC) [reply EpikCord/__init__.py:34]: But sure.

Comment (ZenoABC) [reply EpikCord/__init__.py]: done

**Review by TheUntraceable — APPROVED** [review]

**Comment by TheUntraceable** [comment]
In the issue description, you say "and an issue fix, probably", what is the issue?

**Comment by ZenoABC** [comment]
i doubt it was "issue", but i changed permissions part to use Permissions flag

real issue is not importing humanize lib

**Comment by TheUntraceable** [comment]
Ah makes sense. Thanks for this PR

**Comment by ZenoABC** [comment]
your welcome

**Pull request merged**